site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -4,145 +4,201 @@ name: Interest Invokers (Explainer)
 ```
 
-The `interesttarget` value should be an IDREF pointing to an element within the
-document. `.interestTargetElement` also exists on the element to imperatively
-assign a node to be the invoker target, allowing for cross-root invokers (in
-some cases, see [attr-asociated element
-steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element)
-for more).
-
-Elements with `interesttarget` set will dispatch an `InterestEvent` on the
-_Interestee_ (the element referenced by `interesttarget`) when the element
-_Shows Interest_ or _Loses Interest_. When the element _Shows Interest_ the
-event type will be `'interest'`. If the element has _Shown Interest_, and
-interest moves away from both the _Interest Element_ and the _Interestee_, then
-the element _Loses Interest_ and an `InterestEvent` with the type of
-`'loseinterest'` will be dispatched on the _Interestee_. The event also contains
-a `invoker` property that will reference the _Interest_ element.
-`InterestEvent` objects are always non-bubbling, composed, cancellable events.
+The `interesttarget` value should be an IDREF pointing to an element within a single document or shadow root. The `interestTargetElement` IDL attribute also exists on the element to imperatively assign a node to be the invoker target, allowing for cross-root invokers (in some cases, see [attr-asociated element steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element) for more). We define this target element to be the "Interestee".
+
+Elements with an `interesttarget` attribute will dispatch an `InterestEvent` on the *Interestee* when the element
+*Shows Interest* or *Loses Interest*. When the user *Shows Interest* in the element, the event type will be `"interest"`. If the user has already shown interest in the element, and interest is subsequently lost, an `InterestEvent` with the type of `"loseinterest"` will be dispatched on the *Interestee*. The event also contains a `source` property that will reference the element with `interesttarget`. `InterestEvent` objects are always non-bubbling, composed, cancellable events.
 
 ```webidl
 [Exposed=Window]
 interface InterestEvent : Event {
-  constructor(DOMString type, InterestEventInit interestEventInit);
-  readonly attribute Element invoker;
-  readonly attribute DOMString type = "interest";
-  readonly attribute DOMString action;
+  constructor(DOMString type, optional InterestEventInit interestEventInit = {});
+  readonly attribute Element source;
 };
 dictionary InterestEventInit : EventInit {
-  DOMString action = "";
-  Element invoker;
+  Element? source = null;
 };
 ```
 
-Both `interesttarget` and `commandfor` can exist on the same element at the
-same time, and both should be respected.
+Both `interesttarget` and `commandfor` can exist on the same element at the same time, and both should be respected/functional, since they are activated using strictly separate actions by the user.
```

Comment:
I have an accessibility question related to this, so while I remember.

How does aria-expanded work here? Say you're interested in a tooltip popover, but haven't opened the main dialog that this launches?

Reply:
Good question. I added a mention, since it seems like `aria-expanded` should get set also. But more discussion is needed here. @scottaohara and @aleventhal, FYI and call for help! If this ends up landing before you have a chance to look at the PR, no problem, I'm happy to land followups.